cmd/fyne/internal/commands/bundle.go

```diff
@@ -227,6 +215,32 @@ func (b *Bundler) doBundle(filepath string, out *os.File) {
 	writeResource(filepath, b.name, out)
 }
 
+func getOutputFile(filePath string, noheader bool) (file *os.File, close func() error, err error) {
```

Comment:
Naming here could be improved - "get" methods should have no side effects, but this one has (it will create files). Not inspired, but maybe "createOrAppendFile" is more descriptive?

Reply:
How about:
```suggestion
func openOutputFile(filePath string, noheader bool) (file *os.File, close func() error, err error) {
```

Reply:
Sure